projects/observability/src/shared/components/entity-renderer/entity-renderer.component.ts

```diff
@@ -28,16 +28,16 @@ import { EntityNavigationService } from '../../services/navigation/entity/entity
 })
 export class EntityRendererComponent implements OnChanges {
   @Input()
-  public entity!: Entity;
+  public entity?: Entity;
 
   @Input()
   public navigable: boolean = true;
 
   @Input()
   public showIcon: boolean = false;
 
-  public name!: unknown;
-  public entityIconType!: string;
+  public name?: unknown;
```

Comment:
why not string?

Reply:
Not sure. I just left it as it was, but I can change in my local to string.